source/features/repo-age.tsx

```diff
@@ -123,7 +123,7 @@ void features.add({
 		pageDetect.isRepoRoot
 	],
 	exclude: [
-		pageDetect.isEmptyRepoRoot
+		() => !select.exists('link[rel="canonical"]')
```

Comment:
I think isEmptyRepo should be isRepoHome and :arrow_up:

Reply:
```suggestion
		// Excludes empty repos, compatible with `waitForDomReady: false`
		() => !select.exists('link[rel="canonical"]')
```